packages/td-tools/src/thing-model-helpers.ts

```diff
@@ -453,15 +447,15 @@ export class ThingModelHelpers {
         return tmpThingModels;
     }
 
-    private static getThingModelRef(data: Record<string, unknown>): Record<string, unknown> {
-        const refs = {} as Record<string, unknown>;
+    private static getThingModelRef(data: Record<string, unknown>): Record<string, string> {
```

Comment:
A general comment. Changes to ThingModel code should also go to https://github.com/eclipse-thingweb/td-tools/tree/main/node/thing-model later...

Reply:
I created https://github.com/eclipse-thingweb/td-tools/issues/33